arangod/Agency/Supervision.cpp

```diff
@@ -1817,7 +1819,20 @@ bool arangodb::consensus::cleanupFinishedOrFailedJobsFunctional(
       }
       auto created = p.second->hasAsString("timeCreated");
       if (created) {
-        v.emplace_back(p.first, *created);
+        auto finished = p.second->hasAsString("timeFinished");
+        if (finished) {
+          try {
+            if (std::chrono::system_clock::now() -
+                    stringToTimepoint(finished.value()) >
+                std::chrono::seconds{3600}) {
```

Comment:
Let's make this a named constant similar to `maximalFinishedJobs` above.

Reply:
done